src/smoothscroll.js

```diff
@@ -1,7 +1,11 @@
 'use strict';
 
 // polyfill
-function polyfill() {
+function polyfill(config) {
+  // setup config
+  config = config ? config : {};
+  var { timeCalc } = config;
```

Comment:
I wouldn't open this as a user configuraiton as the spec doesn't provide an official way to change this, and we want to mimic that behavior (even with its limitations).

Reply:
understand, I will make some changes